bioblend/galaxy/workflows/__init__.py

```diff
@@ -123,12 +123,12 @@ def import_workflow_from_local_path(self, file_local_path):
 
         return self.import_workflow_json(workflow_json)
 
-    def import_shared_workflow(self, workflow_id):
+    def import_shared_workflow(self, shared_workflow_id):
```

Comment:
Can you revert the parameter name change here? This may potentially break downstream code using this as a keyword argument.